web-external/src/style/full-viewport.styl

```diff
@@ -1,24 +1,23 @@
-
 html
-  height: 100%
+  height 100%
 
 body
-  position: absolute
-  top: 0
-  bottom: 0
-  left: 0
-  right: 0
+  position absolute
+  top 0
+  bottom 0
+  left 0
+  right 0
 
 #g-global-nav-container
-  z-index: 1
+  z-index 1
 
-#g-app-footer-container
-  position: fixed
-  bottom: 0
-  left: 0
-  right: 0
+DISABLE#g-app-footer-container
+  position fixed
+  bottom 0
+  left 0
+  right 0
 
-#g-app-body-container
-  height: 95%
-  padding-bottom: 88px
+DISABLE#g-app-body-container
+  height 95%
+  padding-bottom 88px
```

Comment:
If the contents of the main panel are too long, they get hidden behind the footer (and I can't scroll to see them).  It might be easier to show you.

This changed what Girder does -- girder has a minimum content size that the footer appears below.  If the content is longer, the footer is off screen initially, and only visible when you scroll to it).  Do you want to permanently consume screen real-estate for the footer instead?